front/src/modules/timesStops/hooks/useTimeStopsColumns.tsx

```diff
@@ -69,30 +80,36 @@ export const useTimeStopsColumns = <T extends TimeStopsRow>(
       {
         ...keyColumn('name', createTextColumn()),
         title: t('name'),
+        ...(isOutputTable && {
+          component: ({ rowData }) => (
+            <span
+              title={rowData.name}
+              className="ml-1 whitespace-nowrap overflow-hidden"
+              style={{ whiteSpace: 'nowrap' }}
```

Comment:
```suggestion
              className="ml-1 text-nowrap overflow-hidden"
```

Reply:
Done, thanks!

I am deeply confused about the difference between text-nowrap and whitespace-nowrap though ^^''